documentation/source/users/arkane/input.rst

```diff
@@ -818,24 +818,28 @@ High Pressure Limit Kinetics
 
 Use a ``kinetics()`` function to make Arkane execute the high-pressure limit kinetic
 parameters computation for a reaction. The ``'label'`` string must correspond to that of
-a defined ``reaction()`` function. If desired, define a temperature range and number
-of temperatures at which the high-pressure rate coefficient will be tabulated and saved to
-the outupt file. The 3-parameter modified Arrhenius coefficients will automatically be fit
-to the computed rate coefficients. The quantum tunneling factor will also be displayed.
+a defined ``reaction()`` function.
 
-Below is a typical ``kinetics()`` function::
+You have three options for specifying the temperature which a modified Arrhenius
+expression will be fit.
+
+Give an explicit list of temperatures to fit::
 
     kinetics(
     label = 'H + C2H4 <=> C2H5',
-    Tmin = (400,'K'), Tmax = (1200,'K'), Tcount = 6,
+    Tlist = ([400,500,700,900,1100,1200],'K'),
```

Comment:
This is misleading since we only take Tmin and Tmax and the count from this list, we don't actually use the intermediate values. Perhaps we should (what do you think?). Either way, the documentation should reflect what we do.

Reply:
I definitely think we should use all the values, which fortunately is what the current implementation does. In `generateKinetics`, we calculate the TST rate at each temperature, shown below.

```
        for i, t in enumerate(self.Tlist.value_si):
            klist[i] = self.reaction.calculateTSTRateCoefficient(t)
```